app/views/defence_requests/show.html.erb

```diff
@@ -31,7 +31,7 @@
       <%= render "defence_requests/show_partials/interview_tab" %>
     </div>
     <div id="case-details">
-      <h3 class="nonjs"><%= t("detention_details")%></h3>
+      <h3 class="nonjs"><%= t("case_details")%></h3>
```

Comment:
This seems like a terrible idea. If one user talks to another having different terminologies will be really confusing...